sirepo_bluesky/__init__.py

```diff
@@ -1,3 +1,13 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+import intake
+
+# Look up a driver class by its name in the registry.
+catalog_class = intake.registry["bluesky-mongo-normalized-catalog"]
+
+sirepo_bluesky_catalog_instance = catalog_class(
+    metadatastore_db="mongodb://localhost:27017/md",
+    asset_registry_db="mongodb://localhost:27017/ar",
+)
```

Comment:
A note for myself: https://github.com/bluesky/databroker/blob/970e9148dfab5e77101d40f059ecb30d064eac81/databroker/_drivers/mongo_normalized.py#L159.